_posts/2023-07-17-13.0.0-release.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "Apache Arrow 13.0.0 Release"
+date: "2023-07-17 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 13.0.0 release. This covers
+over 3 months of development work and includes [**XXX resolved issues**][1]
+from [**YYY distinct contributors**][2]. See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 12.0.0 release, Marco Neumann, Gang Wu, Mehmet Ozan Kabak and Kevin Gurney
+have been invited to be committers.
+Matt Topol, Jie Wen, Ben Baumgold and Dewey Dunnington have joined the
+Project Management Committee (PMC).
+
+Thanks for your contributions and participation in the project!
+
+## Columnar Format Notes
+
+## Arrow Flight RPC notes
+
+## C++ notes
+
+## C# notes
+
+## Go notes
+
+## Java notes
+
+## JavaScript notes
+
+## Python notes
+
+
+## R notes
+
+For more on what’s in the 13.0.0 R package, see the [R changelog][4].
+
+## Ruby and C GLib notes
```

Comment:
@kou @mrkn could you help with the Ruby and C GLib notes for 13.0.0 ?

Reply:
@kou Could you please review the changes below?

```suggestion
## Ruby and C GLib notes

### Ruby

#### Bug fixes

* Fixed GC-related issue against random segfault in hash join [GH-35819](https://github.com/apache/arrow/issues/35819)
* Fixed segfault in `CallExpression.new` [GH-35915](https://github.com/apache/arrow/issues/35915)

#### Improvements

* FlightRPC: Added a convenient wrapper for the authentication method [GH-35435](https://github.com/apache/arrow/issues/35435)
* Added empty table support in `#select_columns` [GH-35681](https://github.com/apache/arrow/issues/35681)
* Added optional hash support in `Expression.try_convert` [GH-35915](https://github.com/apache/arrow/issues/35915)
* Parquet: Added `Parquet::ArrowFileReader#each_row_group` [GH-36008](https://github.com/apache/arrow/issues/36008)
* Added support of the automatic installation of arrow-c-glib on Conda environment [GH-36287](https://github.com/apache/arrow/issues/36287)

### C GLib

#### Bug fixes

* Parquet: Fixed GC-related bug in metadata dependencies [GH-35266](https://github.com/apache/arrow/issues/35266)
* Fixed potentially GC-related issue against random segfault in hash join [GH-35819](https://github.com/apache/arrow/issues/35819)

#### Improvements

* FlightRPC: Added support to pass `GAFlightServerCallContext` object in several methods of `GAFlightServerCustomAuthHandler` [GH-35377](https://github.com/apache/arrow/issues/35377)
* FlightSQL: Added support for INSER/UPDATE/DELETE [GH-36408](https://github.com/apache/arrow/issues/36408)
* Added `GArrowRunEndEncodedDataType` [GH-35417](https://github.com/apache/arrow/issues/35417)
* Added `GArrowRunEndEncodedArray` [GH-35418](https://github.com/apache/arrow/issues/35418)
```

Reply:
@mrkn Thanks!